tests/manager/celery/tasks/test_opds_odl.py

```diff
@@ -556,3 +704,911 @@ def test_wrong_protocol(
         )
         with pytest.raises(ValueError, match=r"is not a OPDS2\+ODL collection"):
             opds_odl.import_collection.delay(collection.id).wait()
+
+    @freeze_time("2016-01-01T00:00:00+00:00")
```

Comment:
These tests are moved over from the `TestOPDS2WithODLImporter` class. They previously tested at the importer level, but I moved them here to test the full import task.